### PR TITLE
Load channel peer suggestions from API

### LINKED ISSUE #294: Use getalby.com API for getting channel suggestions

### PR DESCRIPTION
**Pull request by bumi**
this suggests the user some channels they can connect to.

closes #294 
fixes https://github.com/getAlby/nostr-wallet-connect-next/issues/293

### CODE REVIEW

File: alby/alby.go [hunk 1]
```diff
@@ -44,6 +44,16 @@ type AlbyBalance struct {
 	Currency string `json:"currency"`
 }
 
+type ChannelPeerSuggestion struct {
```
Comment (rolznz) [inline alby/alby.go:47]: @bumi this misses the LSP info, so the lightning options are failing

Comment (rolznz) [reply alby/alby.go:47]: fixed

**Review by rolznz — APPROVED** [review]
tACK

**Comment by bumi** [comment]
@rolznz thanks for fixing it!

**Pull request merged**